components/ontology-access/api/src/main/java/org/phenotips/ontology/internal/solr/AbstractHGNCSolrOntologyService.java

```diff
@@ -0,0 +1,339 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.phenotips.ontology.internal.solr;
+
+import org.phenotips.obo2solr.ParameterPreparer;
+import org.phenotips.obo2solr.TermData;
```

Comment:
You should not create dependencies from this component to the obo2solr package. If some classes from obo2solr are useful here, consider extracting them in another package that is used by both this component and the obo2solr one.